test/test.client.js

```diff
@@ -18,7 +18,7 @@ describe('Client', function () {
     before(function () {
       zk = new FakeZookeeper();
 
-      Client = proxyquire('../lib/Client', {
+      Client = proxyquire('../lib/client.js', {
```

Comment:
You are right @hyperlink - I amended the commit and verified the build is still successful on Travis: https://travis-ci.org/lightswitch05/kafka-node/builds/148936864